src/core/id-headers.js

```diff
@@ -14,7 +14,9 @@ const localizationStrings = {
      * @param {"Appendix" | "Section"} sectionType
      */
     permalinkLabel(sectionType, sectionNumber) {
-      let label = `Permalink for ${sectionType}`;
+      let label = `Permalink for${
+        !sectionNumber ? " this" : ""
```

Comment:
Curious why we might not want "this" for all headings.

Reply:
Its for a numbered section: "For section X.X", VS for a section without a number: "For **this** Section". It makes sense in context.